modules/lightning_features/lightning_media/tests/src/FunctionalJavascript/MediaReferenceFieldTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Drupal\Tests\lightning_media\FunctionalJavascript;
+
+use Drupal\FunctionalJavascriptTests\JavascriptTestBase;
+
+/**
+ * @group lightning
+ * @group lightning_media
+ */
+class MediaReferenceFieldTest extends JavascriptTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $profile = 'lightning';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $strictConfigSchema = FALSE;
```

Comment:
I'm not sure there's any existing one. Also didn't find any issue after doing a [quick look into the issue queue of media_entity_image](https://www.drupal.org/project/issues/media_entity_image?categories=All) Go ahead and create one